### ISSUE TITLE
First page Blank

### ISSUE THREAD

**Issue by ankitchauhan27** [privateOmega/html-to-docx#5]
It is creating docx from HTML string but making first page blank and starting with page 2 in docx. Can you please look into that ?

**Comment by privateOmega** [comment]
@ankitchauhan27 Could you please post the exact html string that you have used?

**Comment by privateOmega** [comment]
Closing the issue due to inactivity.

**Comment by anirudh-modi** [comment]
```
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8" />
        <title>Document</title>
        <style>
        tr{
            color:red;
        }
        </style>
    </head>
    <body>
        <p>Taken from wikpedia</p>
        <img
            src="data:image/png;base64, iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg=="
            alt="Red dot"
        />
        <p>
            <strong>
                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make
                a type specimen book.
            </strong>
            <i>It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.</i>
            <u>It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages,</u>and more recently with desktop publishing software
            <span style="color: hsl(0, 75%, 60%);"> like Aldus PageMaker </span>including versions of Lorem Ipsum.
            <span style="background-color: hsl(0, 75%, 60%);">Where does it come from? Contrary to popular belief, Lorem Ipsum is not simply random text.</span>
            It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old.
        </p>
        <ul style="list-style-type: circle;">
            <li>Unordered list element</li>
        </ul>
        <br>
        <ol style="list-style-type: decimal;">
            <li>Ordered list element</li>
        </ol>
        <br>
        <table style="font-family: arial, sans-serif; border-collapse: collapse; width: 100%;" width="100%">
  <tr>
    <th style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">Company</th>
    <th style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">Contact</th>
    <th style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">Country</th>
  </tr>
  <tr>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px; background-color: #dddddd;" align="left" bgcolor="#dddddd">Alfreds Futterkiste</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px; background-color: #dddddd;" align="left" bgcolor="#dddddd">Maria Anders</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px; background-color: #dddddd;" align="left" bgcolor="#dddddd">Germany</td>
  </tr>
  <tr>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">Centro comercial Moctezuma</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">Francisco Chang</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">Mexico</td>
  </tr>
  <tr>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px; background-color: #dddddd;" align="left" bgcolor="#dddddd">Ernst Handel</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px; background-color: #dddddd;" align="left" bgcolor="#dddddd">Roland Mendel</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px; background-color: #dddddd;" align="left" bgcolor="#dddddd">Austria</td>
  </tr>
  <tr>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">Island Trading</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">Helen Bennett</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">UK</td>
  </tr>
  <tr>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px; background-color: #dddddd;" align="left" bgcolor="#dddddd">Laughing Bacchus Winecellars</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px; background-color: #dddddd;" align="left" bgcolor="#dddddd">Yoshi Tannamuri</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px; background-color: #dddddd;" align="left" bgcolor="#dddddd">Canada</td>
  </tr>
  <tr>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">Magazzini Alimentari Riuniti</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">Giovanni Rovelli</td>
    <td style="border: 1px solid #dddddd; text-align: left; padding: 8px;" align="left">Italy</td>
  </tr>
</table>
        <br>
        <ul>
            <li>
                <strong>
                    <u>Coffee</u>
                </strong>
            </li>
            <li>Tea
                <ol>
                    <li>Black tea
                        <ol>
                            <li>Srilankan <strong>Tea</strong>
                                <ul>
                                    <li>Uva <b>Tea</b></li>
                                </ul>
                            </li>
                            <li>Assam Tea</li>
                        </ol>
                    </li>
                    <li>Green tea</li>
                </ol>
            </li>
            <li>Milk
                <ol>
                    <li>Cow Milk</li>
                    <li>Soy Milk</li>
                </ol>
            </li>
        </ul>
    </body>
</html>
```

@privateOmega By default the generated doc is not showing a blank page, however when I upload the same to slack, I have observed it adds a new blank page for its preview..

**Comment by anirudh-modi** [comment]
[example.docx](https://github.com/privateOmega/html-to-docx/files/5254343/example.docx)
This is the doc which was generated using this library

**Comment by privateOmega** [comment]
@anirudh-modi I have also observed that TBH, but I assumed that was an issue with slack preview parser. I opened this file in Word 2007 and opens without any issues.

**Comment by anirudh-modi** [comment]
Agree.. however there was definitely some other library named `docxjs` although I am aware that they take a different approach and don't convert an HTML to DOCX.... But the output from that library worked fine for slack previews... Could we compare the XML output for both the files? 

Perhaps we may have to add an extra attribute somewhere?

**Comment by privateOmega** [comment]
@anirudh-modi yes even I used to use that. I made this library partially based on that and because of what it lacks, html-docx-js should be the library you are referring to. What it does is basically make use of a ooxml standard feature called alt chunks which allows embedding html and other markup language code snippets directly inside the docx package file. But the problem associated with it despite it looking good on slack preview, is the limited support it provides, eg the file generated based on that altchunk concept won't work in Google docs, WPS, libre office etc. To be precise, no other software than microsoft word for windows and wasnt even supported until later for Microsoft word for mac. Since this project codebase seems twisted in so many ways, I agree it's not a good solution we want, but it works and it's something I felt we need for the time being.
